grr/server/grr_response_server/gui/api_plugins/timeline.py

```diff
@@ -168,10 +173,11 @@ def _GenerateTimeline(
 
     if format == ApiGetCollectedTimelineArgs.Format.RAW_GZCHUNKED:
       filename = f"{client_id}_{fqdn}.gzchunked"
-    elif format == ApiGetCollectedTimelineArgs.Format.BODY:  # pytype: disable=attribute-error
+    elif format == ApiGetCollectedTimelineArgs.Format.BODY:
       filename = f"{client_id}_{fqdn}.body"
     else:
-      message = "Incorrect timeline export format: {}".format(format)
+      message = "Unknown file extension for timeline export format: {}".format(
+          format)
```

Comment:
Nit: if you use f-strings, you can avoid an ugly line break here.

Reply:
Fixed it. Thank you!

Reply:
Hm, actually, now thinking about it: the text is also not correct. It is not the file extension that is unknown, it is the format that has been specified (the format implies the file extension, if known, but the API user has no control over the extension itself).

Reply:
You are right, but the format is already checked in the top-level [Handle method](https://github.com/alexandrucosminmihai/grr/blob/e45d4e12ef92682243f68c03285826d4c30dba78/grr/server/grr_response_server/gui/api_plugins/timeline.py#L114) and I thought that it would be useful for the developer adding new export formats to be notified in this "private" method if they forgot to specify the file extension to be used by the new format. Please let me know if you want me to change it.

Reply:
Oh, you are right. In this case, I would not even write any exception message, just raise `AssertionError` without anything (as this is just an equivalent of "the impossible happened").

Reply:
Changed it. Thank you!